README.md

```diff
@@ -100,41 +315,19 @@ Tweepy is a Python library that allows users to access the Twitter API. We used
 
 In order for the app to function correctly, the user must set up their own environment variables. There should be a .env file containing the following:
 
-🚫These are just examples, replace them with the specifics for your app
```

Comment:
We need a section to tell the user what env variables they need to set up. We won't give them the values, but they need to know which ones exist since they don't get a .env file.

Reply:
I add the env I currently have but we might want to change it